.eslintrc.json

```diff
@@ -32,6 +32,7 @@
     "valid-jsdoc": "off",
     "no-unused-expressions": ["error", { "allowShortCircuit": true }],
     "no-irregular-whitespace": ["error", { "skipTemplates": true }],
+    "no-duplicate-imports": "off",
```

Comment:
This would be necessary wherever we re-export `type` objects, which could be in `src/index.ts` or other "re-export" files.

I could see value in manually excluding these lines as necessary.

Reply:
I've updated the file exclusions to make this more granular.